lib/dogma.ex

```diff
@@ -10,6 +10,8 @@ defmodule Dogma do
   alias Dogma.Rules
   alias Dogma.ScriptSources
 
+  @version Mix.Project.config[:version]
```

Comment:
Nope. It's read at *compile* time for the individual project. I'm using the same "trick" in InchEx's Formatter to add the InchEx version to JSON output: https://github.com/rrrene/inch_ex/blob/99987b500d5f14033645a68efb4fde7f7aa11a7b/lib/inch_ex/docs/formatter.ex#L2